src/encord_active/lib/coco/importer.py

```diff
@@ -14,6 +14,7 @@
 from PIL import ImageOps
 from tqdm.auto import tqdm
 
+from encord_active.cli.config import app_config
```

Comment:
A general comment on imports:
Our app is structured into 3 projects. In the `src/encord_active` folder we have 3 modules, `app` `lib` and `cli`. We structured it this way so we could eventually split it into separate redistributable. In order to allow such thing in the future, we should scope our import to the eventual dependencies between the projects.

`app` depends on `lib`
`cli` depends on `lib` and `app`
`lib` doesn't have any dependencies.

I would suggest getting the ssh key as an argument.

Reply:
That makes sense, removed the offending import